tfjs-core/src/util_test.ts

```diff
@@ -665,3 +665,58 @@ describe('util.decodeString', () => {
     expect(util.isPromise(promise3)).toBeFalsy();
   });
 });
+
+describe('setTimeout', () => {
+  // If we set a larger number here, an error will be reported as "'expect'
+  // was used when there was no current spec, this could be because an
+  // asynchronous test timed out".
+  const totalCount = 8;
+  const skipCount = 5;
```

Comment:
Nit: Document what this does and why it exists.
```suggestion
  // Skip the first few samples because the browser does not throttle them.
  const skipCount = 5;
```

Reply:
comments added